docs/resources/policy_transit_gateway_static_routes.md

```diff
@@ -0,0 +1,81 @@
+---
+subcategory: "beta"
```

Comment:
We normally put here "Beta", IDK if TF doc rendering is case sensitive for this attribute, but maybe it's better to change it instead of finding out at release time.